packages/lmi/src/lmi/embeddings.py

```diff
@@ -88,6 +89,20 @@ class LiteLLMEmbeddingModel(EmbeddingModel):
             " Router is not used here."
         ),
     )
+    _router: litellm.Router | None = None
+
+    @property
+    def router(self) -> litellm.Router:
+        if self._router is None:
+            router_kwargs: dict = self.config.get("router_kwargs", {})
+            if self.config.get("pass_through_router")
```

Comment:
This is a syntax error, need `:`

Reply:
Should be fixed now.

Reply:
Thanks, but can you get `pre-commit` to pass too, it's still failing